### PR TITLE
use device params provided by browser/WebVR API if not using polyfill

### PR DESCRIPTION
**Pull request by jzitelli**
Recently I've experienced weird VR mode when using webvr-boilerplate with Oculus DK2 in WebVR enabled Firefox Nightly or Chromium (fov looks too large or something).  I think it has something to do with setHMDVRDevice parameters setting properties of the non-polyfill HMDVRDevice, which shouldn't be necessary such cases.

### CODE REVIEW

**Comment by brianchirls** [comment]
Also, I often get errors in Chromium in that same spot. It doesn't like that `hmd.setFieldOfView` is receiving objects that aren't instances of the right native constructor. This patch prevents that error.

File: src/webvr-manager.js [hunk 1]
```diff
@@ -471,7 +471,10 @@ WebVRManager.prototype.setHMDVRDeviceParams_ = function(viewer) {
     if (!hmd) {
       return;
     }
-
+    if (hmd.deviceName.indexOf('webvr-polyfill') === -1) {
+      // webvr-polyfill not in use, assume params are provided by browser/WebVR API
```
Comment (borismus) [inline src/webvr-manager.js]: nit: Please capitalize and use periods :)

**Comment by jzitelli** [comment]
No problem, how about this for the comment:

The HMDVRDevice is not provided by WebVR Polyfill, so we assume the browser implements WebVR and has taken care of setting parameters.

I could also move the comment outside of the condition block, that seems to be more consistent with the style.

**Comment by borismus** [comment]
Sounds good to me.

**Comment by brianpeiris** [comment]
+1 FF Nightly on desktop suffers from this.

**Comment by brianpeiris** [comment]
Should probably close this since WebVR 1.0 makes it obsolete.

**Comment by borismus** [comment]
Yep

**Pull request closed without merging**